contracts/ClaimFeesHelper.sol

```diff
@@ -25,7 +33,7 @@ contract ClaimFeesHelper {
     /**
      * @param _booster      Booster.sol, e.g. 0xF403C135812408BFbE8713b5A23a04b3D48AAE31
      * @param _voterProxy   CVX VoterProxy e.g. 0x989AEb4d175e16225E39E87d0D97A3360524AD80
-     * @param _feeDistro    FeeDistro e.g. 0xA464e6DCda8AC41e03616F95f4BC98a13b8922Dc
```

Comment:
shall we update on the code base all references to 0xA464e6DCda8AC41e03616F95f4BC98a13b8922Dc?